R/ggplotly.R

```diff
@@ -176,8 +176,16 @@ gg2list <- function(p){
     df <- df[order(df$order),]
     df$order <- NULL
 
-    misc$prestats.data <- merge(built$prestats.data[[i]],
-                                gglayout[, c("PANEL", "plotly.row", "COL")])
+    prestats <- built$prestats.data[[i]]
+    ## scale_reverse multiples x/y data by -1, so here we undo that so
+    ## that the actual data can be uploaded to plotly.
+    replace.aes <- intersect(names(prestats), reverse.aes)
+    for(a in replace.aes){
```

Comment:
Spaces before `(` and after `)` (see https://google-styleguide.googlecode.com/svn/trunk/Rguide.xml#spacing).